engine/src/main/resources/org/hibernate/validator/ValidationMessages_cs.properties

```diff
@@ -1,27 +1,53 @@
-javax.validation.constraints.AssertFalse.message = mus\u00ed b\u00fdt ne
-javax.validation.constraints.AssertTrue.message  = mus\u00ed b\u00fdt ano
-javax.validation.constraints.DecimalMax.message  = mus\u00ed b\u00fdt men\u0161\u00ed ne\u017e nebo rovno {value}
-javax.validation.constraints.DecimalMin.message  = mus\u00ed b\u00fdt v\u011bt\u0161\u00ed ne\u017e nebo rovno {value}
-javax.validation.constraints.Digits.message      = \u010d\u00edseln\u00e1 hodnota mimo rozsah (o\u010dek\u00e1v\u00e1no <{integer} \u010d\u00edslic>.<{fraction} \u010d\u00edslic>)
-javax.validation.constraints.Email.message       = chybn\u00e1 emailov\u00e1 adresa
-javax.validation.constraints.Future.message      = mus\u00ed b\u00fdt v budoucnosti
-javax.validation.constraints.Max.message         = mus\u00ed b\u00fdt men\u0161\u00ed ne\u017e nebo rovno {value}
-javax.validation.constraints.Min.message         = mus\u00ed b\u00fdt v\u011bt\u0161\u00ed ne\u017e nebo rovno {value}
-javax.validation.constraints.NotBlank.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotEmpty.message    = nem\u016f\u017ee b\u00fdt pr\u00e1zdn\u00fd/\u00e1/\u00e9
-javax.validation.constraints.NotNull.message     = nesm\u00ed b\u00fdt null
-javax.validation.constraints.Null.message        = mus\u00ed b\u00fdt null
-javax.validation.constraints.Past.message        = mus\u00ed b\u00fdt v minulosti
-javax.validation.constraints.Pattern.message     = se mus\u00ed shodovat s "{regexp}"
-javax.validation.constraints.Size.message        = velikost mus\u00ed b\u00fdt mezi {min} a {max}
+javax.validation.constraints.AssertFalse.message     = mus\u00ed m\u00edt hodnotu false (ne)
+javax.validation.constraints.AssertTrue.message      = mus\u00ed m\u00edt hodnotu true (ano)
```

Comment:
Same as Gunnar, I think we should stick to translating true and false. The message is clear enough IMHO.